### PR TITLE
MTE-5652: redo syndicated logic

### PR DESCRIPTION
**Pull request by runrog**
This PR is based off of the podcast PR branch. We should merge that one first.

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

File: src/mock-data.json [hunk 1]
```diff
@@ -233,7 +233,7 @@
       "field_syndicated_content": "True",
       "field_modified_date":"2019-01-22T16:00:55-05:00",
       "field_published_date":"2019-01-09T11:20:57-05:00",
-      "field_tl_": "268:::269:::270:::272:::273",
+      "field_tl_": "268:::269:::270:::272:::273:::1118",
```
Comment (WidgetsBurritos) [inline src/mock-data.json:226]: @runrog we need to remove `field_syndicated_content` and `field_featured_podcast` from these mocks and update any references to them in the code as well, as those fields will get removed from production after this is deployed.

**Review by WidgetsBurritos — CHANGES_REQUESTED** [review]

File: src/mock-categories.json [hunk 2]
```diff
@@ -28,5 +28,10 @@
       "tid":"273",
       "name":"What Are You Solving For?",
       "description":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+   },
+   {
+     "tid": "1118",
+     "name":"What We're Reading... And Our POV On It!",
+     "description":"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
    }
```
Comment (WidgetsBurritos) [inline src/mock-categories.json:41]: As you'll need the `tid` for podcasts, after all, just go ahead and download the latest version of this export here: 
https://admin.rackspace.com/api/thought-leadership-categories?_format=json

File: src/mock-data.json (hunk 1 above)
Comment (runrog) [reply src/mock-data.json:226]: `field_syndicated_content` is in a lot of places including templates/tests FYI. going to take some rebuilding here.

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Comment by WidgetsBurritos** [comment]
Preview URL: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/solve

Login Url: http://mte-5652-redo-syndicated-logic.solve.dev.website.rackspace.com/user

_Note: If your branch name contains underscores or periods, replace them with hyphens prior to attempting to access the links above._

**Review by WidgetsBurritos — APPROVED** [review]
:+1: LGTM

**Pull request merged**